src/services/mock/json/App.json

```diff
@@ -263,7 +263,7 @@
           "finishedTime": "2020-11-08T22:57:41Z",
           "state": "succeeded",
           "submitNum": 4,
-          "messages": ["started", "foo", "bar", "baz", "pub", "qux", "x", "y"],
+          "messages": ["started", "foo message", "bar message", "x", "baz message", "pub message", "qux message", "y"],
```

Comment:
(I moved `x` up so we get a mix of outputs and non-outputs visible in the offline mode; previously `x` and `y` were hidden as beyond the max number shown)